python/pyspark/pandas/tests/test_namespace.py

```diff
@@ -306,16 +329,30 @@ def test_concat_index_axis(self):
 
         objs = [
             ([psdf, psdf], [pdf, pdf]),
+            # no Series
             ([psdf, psdf.reset_index()], [pdf, pdf.reset_index()]),
             ([psdf.reset_index(), psdf], [pdf.reset_index(), pdf]),
             ([psdf, psdf[["C", "A"]]], [pdf, pdf[["C", "A"]]]),
             ([psdf[["C", "A"]], psdf], [pdf[["C", "A"]], pdf]),
+            # only one Series
             ([psdf, psdf["C"]], [pdf, pdf["C"]]),
+            # only one Series
```

Comment:
nit: maybe we can remove this comment here ? The above one should cover.

Reply:
Thanks! Resolved.